src/modules/tests/test-data/cat-c/_tests_/test-data.cat-c.selector.spec.ts

```diff
@@ -0,0 +1,382 @@
+      expect(hasSeriousFault(state, Competencies.controlsClutch)).toBeFalsy();
+    });
+  });
+
+  describe('hasDangerousFault', () => {
+    it('should return true if a competency has a dangerous fault', () => {
+      expect(hasDangerousFault(state, Competencies.useOfSpeed)).toEqual(true);
+    });
+    it('should return false if a competency does not have a dangerous fault', () => {
+      expect(hasDangerousFault(state, Competencies.useOfMirrorsSignalling)).toBeFalsy();
+    });
+  });
+
+  describe('getETAFaultText', () => {
+    it('should return null if no ETA faults', () => {
+      const result = getETAFaultText(state.ETA);
+      expect(result).toBeUndefined();
+    });
+    it('should return `Physical and Verbal` if both ETA faults', () => {
+      state.ETA.physical = true;
+      state.ETA.verbal = true;
+      const result = getETAFaultText(state.ETA);
+      expect(result).toEqual('Physical and Verbal');
+    });
+    it('should return `Physical` if just physical ETA fault', () => {
+      state.ETA.physical = true;
+      state.ETA.verbal = false;
+      const result = getETAFaultText(state.ETA);
+      expect(result).toEqual('Physical');
+    });
+    it('should return `Verbal` if just verbal ETA fault', () => {
+      state.ETA.physical = false;
+      state.ETA.verbal = true;
+      const result = getETAFaultText(state.ETA);
+      expect(result).toEqual('Verbal');
+    });
+  });
+
+  describe('getEcoFaultText', () => {
+    it('should return null if no eco faults', () => {
+      const result = getEcoFaultText(state.eco);
+      expect(result).toBeUndefined();
+    });
+    it('should return `Control and Planning` if both eco faults', () => {
+      state.eco.adviceGivenControl = true;
+      state.eco.adviceGivenPlanning = true;
+      const result = getEcoFaultText(state.eco);
+      expect(result).toEqual('Control and Planning');
+    });
+    it('should return `Control` if just control eco fault', () => {
+      state.eco.adviceGivenControl = true;
+      state.eco.adviceGivenPlanning = false;
+      const result = getEcoFaultText(state.eco);
+      expect(result).toEqual('Control');
+    });
+    it('should return `Planning` if just planning eco fault', () => {
+      state.eco.adviceGivenControl = false;
+      state.eco.adviceGivenPlanning = true;
+      const result = getEcoFaultText(state.eco);
+      expect(result).toEqual('Planning');
+    });
+  });
+
+  describe('getManoeuvres', () => {
+    it('should retrive the manoeuvres data when requested', () => {
+      const result = getManoeuvres(state);
+      expect(result).toEqual(state.manoeuvres);
+    });
+  });
+
+  describe('hasManoeuvreBeenCompleted', () => {
+    it('should return false when no manoeuvres have been completed', () => {
+      const state: CatCUniqueTypes.TestData = {
+        manoeuvres: {},
+      };
+      expect(hasManoeuvreBeenCompletedCatBE(state)).toBeFalsy();
```

Comment:
CatBE - is this awaiting selector rework?

Reply:
Changed it to CatC